### PR TITLE
Enable configuration of org name in reports

### PR DESCRIPTION
**Pull request by nhr**
@thrasher-redhat please review

### CODE REVIEW

**Comment by nhr** [comment]
Shiftzilla now requires 'OrgTitle' configuration item:

screen shot 2019-02-20 at 15 48 55

This is used on the main page of the report:

screen shot 2019-02-20 at 15 48 21

**Comment by thrasher-redhat** [comment]
LGTM.  If you're upping the version, it might be nice to wrap the release names in a string cast as well.

**Comment by thrasher-redhat** [comment]
Side note - will having `OrgTitle` present in the cfg before these changes are made break anything?

File: lib/shiftzilla/release.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ class Release
     attr_reader :name, :targets, :milestones, :default, :token
 
     def initialize(release,builtin=false)
-      @name       = release['name']
+      @name       = release['name'].to_s
```
Comment (nhr) [inline lib/shiftzilla/release.rb:8]: @thrasher-redhat updated PR to force release names to String class. Protects against 'X.Y' version names from being handled as Floats.

**Comment by nhr** [comment]
OrgTitle in config file ahead of time will not break anything. Config file validation does not error on extra keys.

**Review by thrasher-redhat — APPROVED** [review]

**Comment by DirectedSoul1** [comment]
I have tested this latest change. Working for me. Thanks team

**Pull request merged**